tsinfer/cli.py

```diff
@@ -32,7 +32,8 @@
     resource = None  # resource.getrusage absent on windows, so skip outputting max mem
 
 import daiquiri
-import msprime  # for the RateMap class
+
+# import msprime  # for RateMap class: https://github.com/tskit-dev/tsinfer/issues/751
```

Comment:
Just delete this, it's not worth keeping.

Reply:
done